lib/github.js

```diff
@@ -1,39 +1,41 @@
 /* eslint-disable camelcase */
 
-import { log, wasReported } from './log.js';
-import assert from 'assert';
-import fs from 'fs';
-import progress from 'request-progress';
-import request from 'request';
+import assert from "assert";
```

Comment:
Did we change the Prettier confi away from single quotes?

Reply:
Nvm, that's in `pkg`. Let's use the same config.

Reply:
Resolved!